### ISSUE TITLE
Move tabs between windows?

### ISSUE THREAD

**Issue by philipsd6** [osnr/TabFS#71]
I have 300+ tabs open, in 20 different windows. I would like to be able to move tabs between existing tabs, and possibly to new tabs, based on titles, urls, etc... For example:

* "I want to move all my Stack Overflow tabs into a new window to review"
`mv tabs/by-title/*Stack_Overflow* windows/create`

* "I want to move all of my tabs running local apps ("localhost") to the window with my GitHub repo tab open"
`grep -l localhost tabs/by-id/*/url.txt | sed -r 's|/[^/]+$||' | xargs -I{} mv {} tabs/by-title/philipsd6\/dev-project.*/window/`

Is that possible to implement somehow? I've used [balta2ar/brotab](https://github.com/balta2ar/brotab) to accomplish similar goals in the past.

**Comment by osnr** [comment]
I don't think this works right now, but I hope we could make it work -- I'd have to look at what system calls it invokes

(even if we got this to work, in the first example, you'd probably need to create the new window first, _then_ do the move, unless we made a 'fresh window' subfolder)

**Comment by philipsd6** [comment]
Probably not essential to be able to do the first example in one step, but having symmetry with the `create` for a new tab would be nice. I look forward to seeing how this project improves!
